From f82825e00c5dae503f6c132917e0ec134b67423e Mon Sep 17 00:00:00 2001
From: Torge Matthies <openglfreak@googlemail.com>
Date: Mon, 1 Apr 2024 21:47:40 +0200
Subject: [PATCH 36/39] Revert "winegstreamer: Mark decodebin elements as
 streams-aware."

This reverts commit a07bdf9ec0ea2ea130cadbcc8ed81ae88b3b5883.
---
 dlls/winegstreamer/wg_parser.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/dlls/winegstreamer/wg_parser.c b/dlls/winegstreamer/wg_parser.c
index 14a634462e0..f797c54fc0a 100644
--- a/dlls/winegstreamer/wg_parser.c
+++ b/dlls/winegstreamer/wg_parser.c
@@ -715,14 +715,6 @@ static void deep_element_added_cb(GstBin *self, GstBin *sub_bin, GstElement *ele
         g_object_set(element, "n-threads", G_GINT64_CONSTANT(1), NULL);
 #endif
     }
-
-    /* Sometimes, for some reason, an hlsdemux2 gets added to decodebin, but decodebin
-       is not streams-aware, causing hlsdemux to raise an error. */
-    if (name && strstr(name, "Decoder Bin"))
-    {
-        GST_DEBUG("%s found, setting streams-aware flag.", name);
-        GST_OBJECT_FLAG_SET(element, GST_BIN_FLAG_STREAMS_AWARE);
-    }
 }
 
 static gboolean sink_event_cb(GstPad *pad, GstObject *parent, GstEvent *event)
-- 
2.44.0

